PlanningPoker.Entities/Item.cs

```diff
@@ -0,0 +1,20 @@
+using System.ComponentModel.DataAnnotations;
+using System.Collections.Generic;
+
+namespace PlanningPoker.Entities
+{
+    public class Item : BaseEntity
+    {
+        [Required]
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        public ICollection<Round> Rounds { get; set; }
+
+        public Item()
+        {
+            Rounds = new HashSet<Round>();
```

Comment:
Skal der ikke også være en constructor til Item med Title og Desc som parameter? 

Eller er Entities Construction overladt til en anden del?

Maybe I'm overthinking this.

Reply:
Jo, men min tanke var at det lavede man bare ad hoc, som man får brug for det.